source/_includes/asides/hassio_navigation.html

```diff
@@ -8,6 +8,7 @@ <h1 class="title delta">Topics</h1>
           <li>{% active_link /hassio/installation/ Installation %}</li>
           <li>{% active_link /addons/ Available add-ons %}</li>
           <li>{% active_link /hassio/installing_third_party_addons/ Installing third-party add-ons %}</li>
+          <li>{% active_link /hassio/HassOS-FAQ/ Home Assistant OS FAQ %}</li>
```

Comment:
One final thing I missed. Could you please rename the file to contain only lower case characters?